### ISSUE TITLE
Getting started

### ISSUE THREAD

**Issue by ExperiMentor** [dr-mod/pico-solar-system#16]
You're going to think I'm really stupid for this one, but here goes!

I've installed the appropriate UF2 onto my Pico W as instructed, and the Pico W is now connected to my Windows PC running Thonny.

So the next inbstructions are to enter the following - but in what program / device / window or whatever am I supposed to enter them??

pip3 install rshell

git clone https://github.com/dr-mod/pico-solar-system.git

cd pico-solar-system

rshell
cp *.py /pyboard/

**Comment by dr-mod** [comment]
Hi @ExperiMentor , all of the commands you need to execute in your console/terminal (cmd/PowerShell/cigwin/mingw) in Windows. You also need to have pip3 installed in your system for these commands to work. Additionally, you need to make sure that Thonny is not running when you're executing any commands related to rshell.
Hope this helps.

**Comment by ExperiMentor** [comment]
Thanks for your help. I tried those steps, including installing Python 3.11.1, adjusting PATH somit included pip3 script, installed rshell, cloned your repo. But when I run rshell, I get this error:
  File "C:\Program Files\Python311\Lib\site-packages\pyreadline\py3k_compat.py", line 8, in callable
    return isinstance(x, collections.Callable)
                         ^^^^^^^^^^^^^^^^^^^^
AttributeError: module 'collections' has no attribute 'Callable' 
I don't know how to track what is wrong there?

Meanwhile, I discovered that all I really needed to do was download the few repo files and copy them to my Pico W using Thonny. That worked fine, except that the firmware *.UF2 that you link to does not include the required pimoroni and picographics libraries. After finding the correct *.UF2 on the Pimoroni site, I now have a working version of the Orrery. I'm in the process of tweaking it to take advantage of the larger 240 x 240 display that is fitted to my Pimoroni Pico Explorer base.
